examples/wsclient/main.go

```diff
@@ -920,6 +920,7 @@ func callRouteTradeSwap(w *provider.WSClient, ownerAddr string) bool {
 	sig, err := w.SubmitRouteTradeSwap(ctx, &pb.RouteTradeSwapRequest{
 		OwnerAddress: ownerAddr,
 		Project:      pb.Project_P_RAYDIUM,
+		Slippage:     0.01,
```

Comment:
did you intend this to be `0.01`, when the rest of the examples are `0.1`?

Reply:
I don't think it matters but I'll change it